models/topics.go

```diff
@@ -95,10 +95,10 @@ func GetTopicsByCategoryID(categoryID int, before time.Time) []TopicWithUser {
 	return topics
 }
 
-func CreateTopic(categoryID int, userID int, title string, content string, isSticky bool) int {
+func CreateTopic(categoryID int, userID int, title string, content string, isSticky bool, isReadOnly bool) int {
```

Comment:
No need for isReadOnly argument here because we will never create a topic that is closed to start with!

Reply:
Yeah true i had thought about this, but thought it was good to have an option anyway.